src/home.njk

```diff
@@ -5,17 +5,159 @@ permalink: /
+            {# The value of the branch name is dynamically set when opening the dialog #}
+            {# aria-controls value is dynamically set to the id of the wordle row that will be deleted #}
+            <div class="dialog_form_actions">
+                <input type="hidden" name="branchName" value="">
+                <button type="button" id="delete-confirm" aria-controls="" onclick="deleteWordle(this)">
+                    <svg role="presentation">
+                        <use xlink:href="#tick"></use>
+                    </svg>
+                    Yes, delete the question
+                </button>
+                <button type="button" id="delete-cancel" onclick="closeDialog(this)">
+                    <svg role="presentation">
+                        <use xlink:href="#cross"></use>
+                    </svg>
+                    No, keep the question
+                </button>
+            </div>
+        </div>
+    </div>
 </div>
+
+<svg xmlns="http://www.w3.org/2000/svg" style="display: none;">
+    {# An import of src/images/create.svg #}
+    <symbol id="create" viewBox="0 0 18 18" fill="none">
+        <path d="M1 9H17" class="create-svg-class" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+        <path d="M9 1L9 17" class="create-svg-class" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+    </symbol>
+    {# An import of src/images/view-landing.svg #}
+    <symbol id="view" viewBox="0 0 25 15" fill="none">
+        <path d="M1 7.61017C5.71311 13.339 9.86066 14 12.5 14C17.5902 14 21.7377 10.4746 24 7.61017C19.2869 1.88136 15.1393 1 12.5 1C7.22131 1 3.2623 4.9661 1 7.61017Z" class="view-svg-class" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+        <path d="M12.5 12C14.9853 12 17 9.98528 17 7.5C17 5.01472 14.9853 3 12.5 3C10.0147 3 8 5.01472 8 7.5C8 9.98528 10.0147 12 12.5 12Z" class="view-svg-class" stroke-width="1.5" stroke-miterlimit="10" stroke-linecap="round" stroke-linejoin="round"/>
+    </symbol>
+    {# An import of src/images/x.svg #}
+    <symbol id="cross" viewBox="0 0 15 15" fill="none">
+        <path d="M2.00024 2L13.314 13.3137" class="x-svg-class" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+        <path d="M13.3137 2L2.00001 13.3137" class="x-svg-class" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+    </symbol>
+    {# An import of src/images/tick.svg #}
+    <symbol id="tick" viewBox="0 0 22 16" fill="none">
+        <path d="M1 9L7 15L20.5 1" class="tick-svg-class" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
+    </symbol>
+    {# An import of src/images/ellipse.svg #}
+    <symbol id="ellipse" viewBox="0 0 21 21" fill="none">
+        <circle cx="10.5" cy="10.5" r="9.5" stroke="#0080A4" stroke-width="2" stroke-dasharray="4 2"/>
+    </symbol>
+</svg>
 {% endblock %}
 
 {% block headerScripts %}
+<script>
+const globalOptions = {
```

Comment:
Please name this for something related to the application so that people who see it in the global scope have some notion of what it is for. Also would be worth separating out dynamic from static material - i.e. not putting "user" into the same structure. One possibility is the something like the "instance" pattern that is used here - https://github.com/inclusive-design/inverted-wordles/blob/main/src/js/render_wordle.js#L285